docs/reference/stochastic-fictitious-play.rst

```diff
@@ -0,0 +1,99 @@
+.. _stochastic-fictitious-play:
+
+Stochastic fictitious play
+==========================
+
+The stochastic fictitious play algorithm implemented in :code:`Nashpy` is based on the
+one described in [Hofbauer2002]_.
+
+For a game :math:`(A, B)\in\mathbb{R}^{m\times n}` define
+:math:`\kappa_t^{i}:S^{-1}\to\mathbb{N}` to be a function that in a given time
+period :math:`t` for a player :math:`i` maps a strategy :math:`s` from the
+opponent's strategy space :math:`S^{-1}` to a number of total times the opponent
+has played :math:`s`.
+
+As per standard fictitious play, each player assumes their opponent is playing a mixed strategy 
+based on :math:`\kappa_{t-1}`. If no play has taken place, then the probability of playing each 
+action is assumed to be equal. The assumed mixed strategies of a player's opponent are multplied 
+by the player's own payoff matrices to calculate the expected payoff of each action.  
+
+
+A stochastic pertubation :math:`\epsilon_i` is added to each expected payoff :math:`\pi_i` to give a 
+pertubated payoff.  Each :math:`\epsilon_i` is independent of each :math:`\pi_i` and is a random number 
+between 0 and :code:`epsilon_bar`. 
+
+A logit choice function is used to map the pertubated payoff to a non-negative probability distribution, 
+corresponding to the probability with which each strategy is chosen by the player. The logit choice function 
+can be seen below:
+
+.. math::
+
+    L_i( \pi ) = \frac{\exp (\eta ^{-1} \pi_i )}{\sum_{j}\exp (\eta ^{-1} \pi_j)}
+
+Discussion
+----------
+
+Note that similarly to fictitious play, this algorithm will not always converge and sometimes it depends on 
+the form of the game.
+
```

Comment:
I feel this misses the point of stochastic fictitious play no? The whole point is that it will converge "more betterer" than standard fictitious play.